model/src/main/kotlin/Identifier.kt

```diff
@@ -80,8 +80,10 @@ data class Identifier(
         }
     }
 
+    private val list = listOf(provider, namespace, name, version)
```

Comment:
`list` is so generic, what about `components`?

Reply:
I was choosing `list` as that#s the name we're already using in `fromString()`, but I'll change both.